EventFilter/L1TRawToDigi/plugins/implementations_stage2/CaloTowerUnpacker.cc

```diff
@@ -18,7 +18,7 @@ namespace stage2 {
      unsigned int bxid = block.amc().getBX();
 
      // handle offset between BC0 marker and actual BC0...
-     if( (tmt-1) != ((bxid-1+3)%9) ) return true;
+     //if( (tmt-1) != ((bxid-1+3)%9) ) return true;
```

Comment:
hi @rekovic - perhaps this comment needs updating? The functionality it describes I guess is now removed?